### ISSUE TITLE
Menu widgets pop-up causing pillarboxing 

### ISSUE THREAD

**Issue by Keltron3030** [libretro/RetroArch#10541]
## Description
The menu widgets pop-up for retroachievement leaderboards can cause a pillarboxing effect (reverse letterboxing) that will blackout the left and right sides of the screen flanking the notification text and obscuring all parts of the screen that are not directly above or below the notification.  The effect will remain until the game is reset or another leaderboard pop-up occurs.

__Prior to bug__
![MVIMG_20200430_012607](https://user-images.githubusercontent.com/42773860/80676993-fa216f00-8a85-11ea-92c3-4f12af0397e3.jpg)

__During bug__
![MVIMG_20200430_012625](https://user-images.githubusercontent.com/42773860/80677036-19200100-8a86-11ea-8f63-9d145773accf.jpg)

### Steps to reproduce the bug
1. Enable menu widgets: settings>onscreen display>onscreen notifications>graphics widgets on
2. Enable achievements, hardcore mode, leaderboards.
3. Load a game with leaderboards.  I recommend Pac-Man for Atari 2600 as you can quickly trigger leaderboard pop-ups by pressing start once the game starts.

### Bisect Results
This issue seems to have started with the implementation of menu widgets.  I've personally experienced the behavior on 1.8.1+ and I've seen it happen across a variety on cores.

### Version/Commit
- RetroArch: 1.8.5

### Environment information
- OS: Retropie 4.6 (raspbian buster)

**Comment by meleu** [comment]
I confirm this issue.
Also RetroArch 1.8.5 running on RetroPie 4.6.

**Comment by IndegoRune** [comment]
I can also confirm this issue. Happens on a variety of cores, and a variety of games. Only way to stop it is to turn off the notification widgets, and use the text notifications.

**Comment by Keltron3030** [comment]
@natinusala was wondering if you could maybe offer some insight on this issue.

**Comment by natinusala** [comment]
This definitely looks like a bug with scissoring. Does it only happen on RetroPie? Because it might be GPU driver related

**Comment by IndegoRune** [comment]
I can 100% confirm it can happen on lakka, and on retroarch on Windows 10. It's just much much rarer on those.

It seems the worst on a Retropie 4.6 on a Raspberry Pi 3.

It can also happen on a pi 4, but it is much rarer.

![20200502_210047](https://user-images.githubusercontent.com/64575992/81087500-6b668500-8ebf-11ea-9b34-833df2074600.jpg)

Image is if it happening on Lakka for the retroflag GPi.

**Comment by Keltron3030** [comment]
I guess without scissoring, the pop-ups would be a horizontal bar across the whole screen.  Maybe a scissoring on/off config could be a quick band-aid if a fix is elusive.

**Comment by natinusala** [comment]
Scissoring is used to clip the text in the boundaries of the popup frame.

The thing is, if it's so frequent as you say I would have noticed it by the time (widgets are months old), but I've never encountered that problem

**Comment by Keltron3030** [comment]
Yeah, it's pretty strange.  The frequency that it occurs seems to vary widely.

**Comment by SingingCoyote13** [comment]
having the same issue, turning leaderboards off seems to decrease the # times it occurs on my rpi 3b running rpi 4.6.

**Comment by cracklejoy** [comment]
Similar issues under RetroArch 1.8.6 running under rpi 3b RetroPie 4.6.  See attached pics/log.  Not much to see in the logs but the pillarboxing is present.

![mk1-01](https://user-images.githubusercontent.com/65365730/81977853-a91e8880-95f8-11ea-89b1-cd5835a2ca55.png)
![mk1-02](https://user-images.githubusercontent.com/65365730/81977857-a9b71f00-95f8-11ea-9f53-76b387c9973b.png)
![mk1-03](https://user-images.githubusercontent.com/65365730/81977858-a9b71f00-95f8-11ea-8a09-bdc9b1929994.png)
[retroarch__2020_05_14__14_27_23_BAD_PILLARBOX.log](https://github.com/libretro/RetroArch/files/4630482/retroarch__2020_05_14__14_27_23_BAD_PILLARBOX.log)
[retroarch__2020_05_14__14_37_40_GOOD.log](https://github.com/libretro/RetroArch/files/4630483/retroarch__2020_05_14__14_37_40_GOOD.log)
[retroarch__2020_05_14__14_51_17_BAD_PILLARBOX.log](https://github.com/libretro/RetroArch/files/4630484/retroarch__2020_05_14__14_51_17_BAD_PILLARBOX.log)

**Comment by metchebe** [comment]
Hello. I just ran into this in Retroarch 1.8.9 on a Raspberry Pi 3B running Raspbian Buster (Mesa KMS).

**Comment by SingingCoyote13** [comment]
yes cracklejoy THAT is it. precisely - that is what i had on my rp3b  - i have now installed batocera on it (also uses retroarch) and it is reduced by a lot. i just encountered it a couple of times now, but batocera has it less as it seems.
